roles/nvidia-gpu-operator/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Check for DGX packages
+  stat:
+    path: /etc/dgx-release
+  register: is_dgx
+
+- name: Set DGX specific GPU Operator flags
```

Comment:
is this going to work with a heterogeneous set of GPU nodes (DGX and non-DGX)? this looks like it would leave non-DGX nodes without a driver and toolkit

Reply:
No it will not work. Need to figure that out.